### PR TITLE
Translated Lines 9668-9677

### PR DESCRIPTION
**Pull request by HelpfulKoreanGirl**
Also edited a typo on line 8433.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9665,16 +9665,16 @@ QUEST_LV_0100_20151016_009660
...
+QUEST_LV_0100_20151016_009666	They're my research records, but the monsters took off with them for some reason.{nl} I'm going out of my mind trying to figure out what's missing.{nl}
 QUEST_LV_0100_20151016_009667	
-QUEST_LV_0100_20151016_009668	
-QUEST_LV_0100_20151016_009669	
-QUEST_LV_0100_20151016_009670	
-QUEST_LV_0100_20151016_009671	
-QUEST_LV_0100_20151016_009672	
-QUEST_LV_0100_20151016_009673	
+QUEST_LV_0100_20151016_009668	Let's see..{nl} This one's here.. That one's not here.. Now where's that one..
+QUEST_LV_0100_20151016_009669	Oh, Yes.{nl} You've found them all.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Oh, Yes -> Oh, yes.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9665,16 +9665,16 @@ QUEST_LV_0100_20151016_009660
...
+QUEST_LV_0100_20151016_009670	{nl}What was it you wanted to ask me about?
+QUEST_LV_0100_20151016_009671	Albinas?{n1} I haven't heard that name before.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: I haven't heard of that name before.

{n1} -> {nl}

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**